phoenix-core/src/main/java/org/apache/phoenix/coprocessor/GlobalIndexRegionScanner.java

```diff
@@ -197,4 +230,17 @@ protected static long getMaxTimestamp(Pair<Put, Delete> pair) {
         }
         return (ts1 > ts2) ? ts1 : ts2;
     }
+
+    protected boolean isColumnIncluded(Cell cell) {
```

Comment:
Perhaps move to a utility class? CellUtil or some other?

Reply:
I agree with you on this. There are several other such methods that should be moved to utility classes. I suggest opening a Jira for this and doing it there as I want to control the size of the changes here. It is already getting large